src/main/resources/SnykBundle.properties

```diff
@@ -0,0 +1,3 @@
+snyk.settings.organization.tooltip.description=An organization groups projects, and can have team members who can access these projects.
```

Comment:
is some text missing here: `...projects, and...` ?

Reply:
no, the text is from Snyk doc 
<img width="769" alt="image" src="https://user-images.githubusercontent.com/60606414/159179081-35ff0779-f745-4656-bb4c-fe8a4ae62757.png">